Formula/gnupg@2.0.rb

```diff
@@ -79,7 +71,17 @@ def post_install
   end
 
   test do
-    Gpg.create_test_key(testpath)
```

Comment:
It may work again once @JCount's PR goes through https://github.com/Homebrew/brew/pull/2343